ChickenVision/src/org/team1540/chickenvision/Target.java

```diff
@@ -0,0 +1,5 @@
+package org.team1540.chickenvision;
+
+public class Target {
+	
```

Comment:
It will be the class that figures out where the target is. I shouldn't have committed it but it will be filled in in the next commit.